### ISSUE TITLE
Fail to build image for Dockerfile with the default 7.0 image

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-docker#3512]
**OS:** Win 10
**Docker Extension Build Version:** 20220506.1
**.NET Version:** 7.0.100-preview.3.22179.4

**Repro Steps:**
1. Create and open a .NET 7.0 MVC project in VS Code.
2. Add Docker files to this project (Linux Container) -> Right click the "Dockerfile" file -> Select "Build Image...".
3. Check whether succeeds to build image of the "Dockerfile" file.

**Expect:**
Succeed to build image of the "Dockerfile" file.

**Actual:**
Fail to build image of the "Dockerfile" file.
![image](https://user-images.githubusercontent.com/45614674/167351361-ae63bb52-58ed-42d7-9a6d-c38cb126d744.png)

**More Info:**
1. This issue only reproduces with the default Docker image "7.0-focal".
2. This issue does not reproduce with other images "7.0 and 7.0-alpine".
3. This issue does not reproduce for a .NET 6.0 project.

**Comment by bwateratmsft** [comment]
We have updated the scaffolding to return to just `mcr.microsoft.com/dotnet/aspnet:7.0`. It will no longer do the erroneous `mcr.microsoft.com/dotnet/aspnet:7.0-focal`. This applies to all .NET scaffolding including 6.0 and 3.1.

In the past we found that the Debian images (e.g. `:6.0`) were lacking in some binaries needed to support the attach scenario, but the Ubuntu images (e.g. `:6.0-focal`) did have them, so we changed to those. It seems the C# extension has changed some things and those binaries are no longer needed, so we have returned to the Debian images.
